src/ray/core_worker/context.cc

```diff
@@ -0,0 +1,104 @@
+
+#include "context.h"
+
+namespace ray {
+
+/// per-thread context for core worker.
+struct WorkerThreadContext {
+  WorkerThreadContext(WorkerType worker_type, const DriverID &driver_id)
+    : task_index(0),
+      put_index(0) {
+
+    auto initial_driver_id = driver_id;
+    if (initial_driver_id.is_nil()) {
+      initial_driver_id = DriverID::from_random();
+    }
+  
+    if (worker_type == WorkerType::DRIVER) {
+      current_task_id = TaskID::from_random();
```

Comment:
`from_random` was renamed to `FromRandom` earlier today. So are other methods in ID.

Reply:
I see. thanks for the pointer.